core/src/main/java/org/mule/model/resolvers/CallableEntryPointResolver.java

```diff
@@ -43,6 +48,7 @@ public InvocationResult invoke(Object component, MuleEventContext context) throw
     {
         if (component instanceof Callable)
         {
+            copySessionProperties(context.getSession(), context.getMessage());
```

Comment:
If properties need to be copied, that should happen inside the MuleEventContext instance creation, not here